Error installing - can't find 'bedrock-injector' module

Fresh install on Node 8.9.1.
Contents of `/etc/hosts`:

```
127.0.0.1 localhost
127.0.0.1 genesis.veres.one.localhost peer-1.veres.one.localhost peer-2.veres.one.localhost peer-3.veres.one.localhost
```

Error on npm install:

```
> veres-one@0.0.1 postinstall ~/_DigitalBazaar/veres-one
> node genesis.veres.one.localhost.js compile-less

module.js:538
    throw err;
    ^

Error: Cannot find module 'bedrock-injector'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (~/_DigitalBazaar/veres-one/node_modules/bedrock-ledger-node/lib/injector.js:4:23)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
(node:90468) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGINT listeners added. Use emitter.setMaxListeners() to increase limit
npm ERR! code ELIFECYCLE
```

I suspect the cause lies in me not knowing what Step 3 entails, 'Setup the Veres One development environment.'. Should we link developers to instructions?

--- a/package.json
+++ b/package.json
@@ -31,6 +31,7 @@
     "bedrock-i18n": "^2.0.1",
     "bedrock-identity": "^4.6.0",
     "bedrock-identity-http": "^1.1.0",
+    "bedrock-injector": "^1.0.0",
     "bedrock-issuer": "^1.0.8",
     "bedrock-jobs": "^2.0.3",
     "bedrock-key": "^4.1.0",

